drools-game-engine-services/src/test/java/org/drools/game/services/endpoint/test/StoreFactsTestImp.java

```diff
@@ -0,0 +1,68 @@
+package org.drools.game.services.endpoint.test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.enterprise.context.ApplicationScoped;
+
+import org.drools.game.capture.flag.cmds.CommandRegistry;
+import org.drools.game.capture.flag.model.Chest;
+import org.drools.game.capture.flag.model.Flag;
+import org.drools.game.capture.flag.model.Location;
+import org.drools.game.capture.flag.model.NamedLocation;
+import org.drools.game.capture.flag.model.Team;
+import org.drools.game.capture.flag.model.Zone;
+import org.drools.game.core.api.StoreFacts;
+import org.drools.game.model.api.Player;
+import org.drools.game.model.impl.base.BasePlayerImpl;
+
+
+@ApplicationScoped
+public class StoreFactsTestImp implements StoreFacts{
+	
+	
+	public StoreFactsTestImp() {
+        CommandRegistry.set( "TELEPORT_CALLBACK", "org.drools.game.services.endpoint.test.cmds.TeleportPlayerCommand" );
```

Comment:
@Salaboy  Does this initialization belong to GameConfiguration? I don't know where it should go. 
As far as I know it can't be part of the setupCmds method (in the test class) since they run in different threads.